encoders/src/main/scala/org/apache/spark/sql/execution/columnar/encoding/ColumnDeltaEncoder.scala

```diff
@@ -675,7 +670,7 @@ object DeltaWriter {
         val evaluator = new CompilerFactory().newScriptEvaluator()
         evaluator.setClassName("io.snappydata.execute.GeneratedDeltaWriterFactory")
         evaluator.setParentClassLoader(getClass.getClassLoader)
-        evaluator.setDefaultImports(defaultImports: _*)
+        evaluator.setDefaultImports(defaultImports)
```

Comment:
revert; new parboiled version need the original way

Reply:
Update core/ spark version to 3.0.8.. keeping similar to upstream spark and keep the new change.